### PR TITLE
Adding dvc-objects

### PR DESCRIPTION
**Pull request by efiop**
This will be used in https://github.com/conda-forge/dvc-feedstock

Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by shcheklein** [comment]
I agree to be a maintainer :)

**Comment by efiop** [comment]
As instructed in the PR template, pinging @conda-forge/help-python for review 🙂

**Comment by efiop** [comment]
https://github.com/orgs/conda-forge/teams/staged-recipes Hey folks, this is ready for review 🙏

File: recipes/dvc-objects/meta.yaml
Comment (BastianZim) [inline recipes/dvc-objects/meta.yaml:21]: Upstream also lists `jaraco.windows`. Since you're also upstream, is that no required?

**Review by BastianZim — CHANGES_REQUESTED** [review]
Just one small thing, then this is ready to go 🚀

File: recipes/dvc-objects/meta.yaml
Comment (efiop) [reply recipes/dvc-objects/meta.yaml:21]: @BastianZim Nice catch! I've updated the PR: dropped python 3.7 so we don't even need `jaraco.windows` anymore. Part of the upcoming 3.7 drop in DVC https://github.com/iterative/dvc/issues/7708 . Thank you 🙏

**Review by BastianZim — APPROVED** [review]

**Comment by BastianZim** [comment]
Looks good. Please ping me once the CI passes on Win as well and I’ll merge it.

**Comment by efiop** [comment]
@BastianZim Good to go. Thank you 🙂

**Pull request merged**
